code/__defines/culture.dm

```diff
@@ -2,10 +2,12 @@
 #define TAG_HOMEWORLD "home_system"
 #define TAG_FACTION   "faction"
 #define TAG_RELIGION  "religion"
+#define TAG_CITIZENSHIP "citizenship"
 
 var/global/list/ALL_CULTURAL_TAGS = list(
 	TAG_CULTURE =   "Culture",
-	TAG_HOMEWORLD = "Residence",
+//	TAG_HOMEWORLD = "Residence",
+	TAG_CITIZENSHIP = "Citizenship",
```

Comment:
Literally, "Remove residence from character setup and add citizenship", basically.